tests/qos/test_buffer.py

```diff
@@ -978,7 +981,10 @@ def test_change_speed_cable(duthosts, rand_one_dut_hostname, conn_graph_facts, p
 
             # Re-add another lossless priority
             logging.info('Re-add a lossless_pg and check pool size and pfc_enable')
-            duthost.shell('config interface buffer priority-group lossless add {} 6'.format(port_to_test))
+            if duthost.facts['asic_type'] == 'barefoot':
+                duthost.shell('config interface buffer priority-group lossless add {} 5'.format(port_to_test))
```

Comment:
There are multiple places in this file where this asic check is done to differentiate between lossless pg 5 and 6. Please have this specified as a variable in the dynamic_buffer.json file for each vendor. Then this asic check can be skipped and the variable can be used directly

Reply:
I changing it to stick with the already used pg 6, so less mess will be introduced, thanks. Just need to edit buildimage PR also.